source/pom.xml

```diff
@@ -36,6 +36,12 @@
     <java.version>17</java.version>
 
     <amazons3.version>2.20.56</amazons3.version>
+
+    <sonar.projectKey>aerius_file-server</sonar.projectKey>
+    <sonar.moduleKey>${project.artifactId}</sonar.moduleKey>
+    <sonar.organization>aerius</sonar.organization>
+    <sonar.host.url>https://sonarcloud.io</sonar.host.url>
```

Comment:
Perhaps we could move these to the root pom?

Reply:
Yeah would work as well. @Hilbrand thoughts?